client/components/webcamModule.js

```diff
@@ -0,0 +1,114 @@
+angular.module('in-your-face')
+.controller('webcamModuleCtrl', function($scope) {
+  console.log("123");
```

Comment:
PR states, complete camera integration, is this complete/draft? so need to put meaningful log messages.